assets/src/settings-page/settings-footer.js

```diff
@@ -62,6 +65,10 @@ export function SettingsFooter() {
 	const isBusy = savingOptions || downloadingTheme || savingDeveloperToolsOption;
 	const disabled = ! hasChanges || isBusy || ! themeSupport || ( 'reader' === themeSupport && ! readerTheme );
 
+	const shouldShowPageCacheFlushNotice = (
+		( 'object' === typeof modifiedOptions && ( modifiedOptions.theme_support || modifiedOptions.reader_theme ) )
+	);
```

Comment:
I'm guessing this could be simplified using optional chaining:

```suggestion
	const shouldShowPageCacheFlushNotice = ( modifiedOptions?.theme_support || modifiedOptions?.reader_theme );
```